content/v2.0/write-data/_index.md

```diff
@@ -70,7 +71,8 @@ InfluxDB accepts the following precisions:
 - `ms` - Milliseconds
 - `s` - Seconds
 
-_For more details about line protocol, see the [Line protocol reference](/v2.0/reference/syntax/line-protocol) and [Best practices for writing data](/v2.0/write-data/best-practices/)._
+_For more details about line protocol, see the [Line protocol reference](/v2.0/reference/syntax/line-protocol)
+and [Best practices for writing data](/v2.0/write-data/best-practices/)._
 
 ## Ways to write data into InfluxDB
 
```

Comment:
On line 84 add link: - [Other ways to write data](#other-ways-to-write-data)

Thoughts on making lines 81-84 tabs?: UI | CLI | API | Other

Reply:
The bad thing about tabs is that you can't link directly to headers that are not in the first tab section. They are hidden when the page loads. I think the appropriate use of tabs is when the content only slightly changes between the variations.